Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 // MARK: - Setup
 
-private let environment: Environment = .development
+private let environment: Environment = .production
```

Comment:
.production works good for development? or this is for easier integration in whitelabel?

Reply:
Its for integration in to WL because we're about to use this like production 🙌